core/config/project_settings.cpp

```diff
@@ -375,6 +375,33 @@ Variant ProjectSettings::get_setting_with_override(const StringName &p_name) con
 		WARN_PRINT("Property not found: " + String(name));
 		return Variant();
 	}
+
+	bool getenv_enabled = true;
+#ifndef DEBUG_ENABLED
```

Comment:
Lets just make this check, instead of debug non debug just lets make this check every build type, who cares if its debug or release